### ISSUE TITLE
Building with `ghc-lib-parser-9.0.*`

### ISSUE THREAD

**Issue by re-xyr** [haskell/stylish-haskell#391]
`stylish-haskell-0.14.0.0` supports GHC 9.0 & 9.2 now, but it's difficult to integrate it into HLS for GHC 9.0 because it uses `ghc-lib-parser-9.2.*` and `ghc-lib-parser-ex-9.2.*`, while other formatters still use `ghc-lib-parser-9.0.*`. It'll be nice if there could be a compile flag that allows `stylish-haskell` to build against `ghc-lib-parser-9.0.*` and `ghc-lib-parser-ex-9.0.*`.

**Comment by jaspervdj** [comment]
Agreed.  Porting from 8.x to 9.2 took a long time, but I expect the differences between 9.0 and 9.2 to be smaller.
